torch/jit/__init__.py

```diff
@@ -789,6 +790,95 @@ def wrap_check_inputs(check_inputs):
 
     return [{'forward' : c} for c in check_inputs]
 
+
+def freeze(mod, preserved_attrs : Optional[List[str]] = None):
+    r"""
+    Freezing a :class:`ScriptModule` will clone it and attempt to inline the cloned
+    module's submodules, parameters, and attributes as constants
+    in the TorchScript IR Graph. By default, `forward` will be preserved,
+    as well as attributes specified in `preserved_attrs`. Additionally, attributes which
```

Comment:
Probably should clarify that methods can be specified in `preserved_attrs`. I think many people might conceptualize "attribute" and "method" as being disjoint (because of other OOP systems with non-first-class methods)